### PR TITLE
Component check

### PR DESCRIPTION
**Pull request by sammy-k**
https://trello.com/c/EYFFQZiB

### CODE REVIEW

File: .DS_Store
Comment (daleinen7) [inline .DS_Store:1]: I have no idea why these .DS_Store pages are showing up. See if you can try to remove them. I am running into the same thing with other projects. Must have bene an OSX update.

File: src/pages/components.jsx [hunk 1]
```diff
@@ -185,6 +179,89 @@ const Components = () => {
...
+          eyebrowText={'Eyebrow text goes here'}
+          mainHeading={'Main heading goes here'}
+          children
+          background
+        />
+      </StyledSection>
+
+      <StyledSection className="width-wrapper">
+        <h3>Footer</h3>
+        <Footer />
```
Comment (daleinen7) [inline src/pages/components.jsx:227]: Since the footer is on the page already, let's not include this.

File: src/pages/components.jsx [hunk 2]
```diff
@@ -185,6 +179,89 @@ const Components = () => {
...
+      </StyledSection> */}
+
+      <StyledSection className="width-wrapper">
+        <h3>Image Wrapper</h3>
+        <ImageWrapper alt={'Image description'} image={TestImage} />
+      </StyledSection>
+
+      <StyledSection className="width-wrapper">
+        <h3>Nav</h3>
+        <Nav />
```
Comment (daleinen7) [inline src/pages/components.jsx:243]: Since the navs are technically on the page anyway, let's not include them.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Looks good. 

Just a few minor change requests and it's good to go. Thanks!

File: .DS_Store
Comment (sammy-k) [reply .DS_Store:1]: strange - they arent showing up on my local branch.  Probably something we should look into

**Review by daleinen7 — APPROVED** [review]
@sammy-k Looks good except it's saying the conflict is with the `.DS_Store`.

**Comment by daleinen7** [comment]
> @sammy-k Looks good except it's saying the conflict is with the `.DS_Store`.

@sammy-k I _think_ all you need to do is add the .DS_Store file to the gitignore and it should resolve the conflict. I hope anyway.

**Pull request closed without merging**